apps/serialization/messages.proto

```diff
@@ -69,10 +69,18 @@ message Configuration {
   ClientConfig client_config = 4;
 }
 
+enum GameMode{
+  BATTLE = 0;
+  DEATHMATCH = 1;
+  PAIR = 2;
+  QUICK_GAME = 3;
+}
```

Comment:
Remember to update arena's version. We do that everytime we touch the proto file (and other cases).

Reply:
Sure!